LECT_2/Lect2_homework/job1/helpers.py

```diff
@@ -0,0 +1,42 @@
+import os
+import shutil
+import json
+import requests
+
+AUTH_TOKEN = os.environ.get("API_AUTH_TOKEN")       # Secret token from the environment
+URI = 'https://fake-api-vycpfa6oca-uc.a.run.app/'   # Path to API
+
+
+def get_sales(date, raw_dir=''):
+    page = 1
+    actual_raw_dir = os.getcwd().replace('\\job1', '') + '\\file_storage' + raw_dir
```

Comment:
Your job will work in Windows environment only because you use Windows path separators: `\`. 
Nobody use Windows as a execution environment in production, but if you want to implement a cross-environment job, you should use `os.sep` instead of hardcoded separators. 
Also, if you youse `os.path.join(...)` to concatenate the paths, it should care about it automatically.